tests/flow/test_persistency.py

```diff
@@ -62,7 +62,7 @@ def populate_graph(self, graph_name):
         actual_result = redis_con.execute_command(
                 "GRAPH.QUERY", graph_name, "CREATE INDEX FOR ()-[r:visit]-() ON (r.purpose)")
         actual_result = redis_con.execute_command(
-                "GRAPH.QUERY", graph_name, "CALL db.idx.fulltext.createNodeIndex({label: 'person', stopwords: ['A', 'B'], language: 'english'}, 'text')")
+                "GRAPH.QUERY", graph_name, "CALL db.idx.fulltext.createNodeIndex({label: 'person', stopwords: ['A', 'B'], language: 'english'}, { field: 'text', nostem: true, weight: 2, phonetic: 'dm:en' })")
```

Comment:
Can't we create a full-text index using:
`CALL db.idx.fulltext.createNodeIndex({label: 'person', stopwords: ['A', 'B'], language: 'english'}, 'text')` ?
If so why did you changed this line ?

Reply:
we need to check that the options is saved(for now can't check if they are because db.indexes not returning this data)